src/Analysis/Ast/Impl/Resources.resx

```diff
@@ -204,4 +204,7 @@
   <data name="GenericNotAllUnique" xml:space="preserve">
     <value>Arguments to Generic must all be unique.</value>
   </data>
-</root>
+  <data name="ComparisonWithCallable" xml:space="preserve">
+    <value>Comparing against a callable, did you omit the parenthesis?</value>
```

Comment:
This is weird. No other message are phrased like this. Perhaps "Functions are not comparable.".

Note that I would also change this to `FunctionComparison` and `function-comparison`, because something can be "Callable" as well as comparable. "Callable" just means it has `__call__`.

Reply:
Ok, sure sounds good.